### PR TITLE
Improvement to create cluster page

### PR DESCRIPTION
**Pull request by RA489**
Improvement for k8s.io/docs/tutorials/kubernetes-basics/create-cluster/ 

fixes https://github.com/kubernetes/website/issues/28886

### CODE REVIEW

File: content/en/docs/tutorials/kubernetes-basics/create-cluster/_index.md [hunk 1]
```diff
@@ -2,3 +2,5 @@
 title: Create a Cluster
 weight: 10
 ---
+
+{{< glossary_definition term_id="cluster" length="short" >}}
```
Comment (jihoon-seo) [inline content/en/docs/tutorials/kubernetes-basics/create-cluster/_index.md:6]: In spite of adding this line, the description for "cluster" does not appear.

---

[As-is]
https://kubernetes.io/docs/tutorials/kubernetes-basics/create-cluster/
image

---

[Preview of this PR]
https://deploy-preview-28905--kubernetes-io-main-staging.netlify.app/docs/tutorials/kubernetes-basics/create-cluster/
image

**Review by jihoon-seo — CHANGES_REQUESTED** [review]
@RA489 PTAL!

File: content/en/docs/tutorials/kubernetes-basics/create-cluster/_index.md (hunk 1 above)
Comment (RA489) [reply content/en/docs/tutorials/kubernetes-basics/create-cluster/_index.md:6]: Thanks for the reviews! I will update accordingly.

Comment (RA489) [reply content/en/docs/tutorials/kubernetes-basics/create-cluster/_index.md]: Updated the PR. PTAL.

**Review by celestehorgan — COMMENTED** [review]
/lgtm

**Comment by jihoon-seo** [comment]
@RA489 In spite of your newer commit and force-push, the description for "cluster" does not appear. 🤔

---

[As-is]
https://kubernetes.io/docs/tutorials/kubernetes-basics/create-cluster/
image

---

[Preview of this PR]
https://deploy-preview-28905--kubernetes-io-main-staging.netlify.app/docs/tutorials/kubernetes-basics/create-cluster/
image

File: content/en/docs/tutorials/kubernetes-basics/create-cluster/_index.md [hunk 2]
```diff
@@ -2,3 +2,5 @@
 title: Create a Cluster
 weight: 10
 ---
+
+{{< glossary_definition term_id="Cluster" length="short" >}}
```
Comment (kbhawkey) [inline content/en/docs/tutorials/kubernetes-basics/create-cluster/_index.md:6]: Hi @RA489 .
Is the `term_id` attribute "cluster"?

Comment (RA489) [reply content/en/docs/tutorials/kubernetes-basics/create-cluster/_index.md:6]: `term_id` attribute "cluster" doesn't render its definition.

Comment (kbhawkey) [reply content/en/docs/tutorials/kubernetes-basics/create-cluster/_index.md:6]: Hi @RA489 .
I ran a test of the glossary_definition shortcode in PR #29712.
Can you modify the glossary definition cluster attribute and test in your PR?

Comment (RA489) [reply content/en/docs/tutorials/kubernetes-basics/create-cluster/_index.md:6]: sure.

Comment (RA489) [reply content/en/docs/tutorials/kubernetes-basics/create-cluster/_index.md]: modified the glossary definition cluster attribute

**Comment by RA489** [comment]
@kbhawkey updated the PR. PTAL.

**Comment by jihoon-seo** [comment]
[Preview of 4fd6a6a]
https://deploy-preview-28905--kubernetes-io-main-staging.netlify.app/docs/tutorials/kubernetes-basics/create-cluster/

---

image

**Comment by RA489** [comment]
lookin for lgtm!

**Comment by shannonxtreme** [comment]
Instead of just having the glossary definition of cluster, why not make this a sentence or two about what this section of our tutorials teaches you? 

Something like:

Learn about Kubernetes [clusters]\(glossary_tooltip here) and create a simple cluster using Minikube.

---

And that's it. The next page explains clusters in depth as well.

**Comment by RA489** [comment]
> Instead of just having the glossary definition of cluster, why not make this a sentence or two about what this section of our tutorials teaches you?
> 
> Something like:
> 
> Learn about Kubernetes [clusters](glossary_tooltip here) and create a simple cluster using Minikube.
> 
> And that's it. The next page explains clusters in depth as well.

SGTM!!

**Comment by jimangel** [comment]
@RA489 are you still working on this? I agree with @shannonxtreme's comment for adjustment. I'm actively cleaning up lingering PRs, so I'll close this one, but if you're still working on it, feel free to reopen it or reach out on [SIG Docs' slack](https://kubernetes.slack.com/messages/sig-docs). Thanks!

/close

**Comment by RA489** [comment]
> @RA489 are you still working on this?

updated the PR. PTAL.

File: content/en/docs/tutorials/kubernetes-basics/create-cluster/_index.md [hunk 3]
```diff
@@ -2,3 +2,6 @@
 title: Create a Cluster
 weight: 10
 ---
+
+Learn about Kubernetes {{< glossary_tooltip text="cluster" term_id="cluster" length="all" >}} and create a simple cluster using Minikube.
```
Comment (sftim) [inline content/en/docs/tutorials/kubernetes-basics/create-cluster/_index.md:6]: nit:
```suggestion
Learn about Kubernetes {{< glossary_tooltip text="clusters" term_id="cluster" length="all" >}} and create a simple cluster using Minikube.
```

**Review by sftim — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**